README.md

```diff
@@ -140,137 +209,244 @@ Let's have a look on the Flume event headers:
+    +------------+-----------------------------+----------+------------+-------------+------------+-----------+--------+
+    | 1429535775 | 2015-04-20T12:13:22.41.124Z | car1     | car        |  speed      | kmh        | 112.9     | []     |
+    | 1429535775 | 2015-04-20T12:13:22.41.124Z | car1     | car        |  oil_level  | percentage | 74.6      | []     |
+    +------------+-----------------------------+----------+------------+-------------+------------+-----------+--------+
+    2 row in set (0.00 sec)
+    
+If `attr_persistence=colum` then `OrionHDFSSink` will persist the data within the body as:
+
+    $ mysql -u myuser -p
+    Enter password: 
+    Welcome to the MySQL monitor.  Commands end with ; or \g.
+    ...
+    mysql> show databases;
+    +-----------------------+
+    | Database              |
+    +-----------------------+
+    | information_schema    |
+    | vehicles              |
+    | mysql                 |
+    | test                  |
+    +-----------------------+
+    4 rows in set (0.05 sec)
+
+    mysql> use vehicles;
+    ...
+    Database changed
+    mysql> show tables;
+    +--------------------+
+    | Tables_in_vehicles |
+    +--------------------+
+    | 4wheels_car1_car   |
+    +--------------------+
+    1 row in set (0.00 sec)
+
+    mysql> select * from 4wheels_car1_car;
+    +-----------------------------+-------+----------+-----------+--------------+
+    | recvTime                    | speed | speed_md | oil_level | oil_level_md |
+    +-----------------------------+-------+----------+-----------+--------------+
+    | 2015-04-20T12:13:22.41.124Z | 112.9 | []       |  74.6     | []           |
+    +-----------------------------+-------+----------+-----------+--------------+
+    1 row in set (0.00 sec)
+
+NOTE: `mysql` is the MySQL CLI for querying the data.
 
-On the contrary, being the persistence mode `column`, the table named `workingrooms_floor4_room1_Room` (it must be created in advance) will contain a new row such as:
+[Top](#top)
 
-    | recvTime               | temperature | temperature_md                                     | 
-    |------------------------|-------------|----------------------------------------------------|
-    | 2014-02-27T14:46:21Z | 26.5        | [{"name":"ID", "type":"string", "value":"ground"}] |
+####<a name="section3.4.4"></a>OrionMongoSink persistence
 
-Each organization/tenant is associated to a different database.
+MongoDB organizes the data in databases that contain collections of Json documents. Such organization is exploited by [`OrionMongoSink`](doc/desing/OrionMongoSink.md) each time a Flume event is taken from its channel.
 
-[Top](#top)
+Assuming `mongo_username=myuser` as configuration parameter, the data within the body will be persisted as:
 
-###<a name="section3.4"></a>OrionMongoSink persistence
-Then `OrionMongoSink` will persist the data within the body as:
-
-    $ mongo
+    $ mongo -u myuser -p
```

Comment:
Yes, it is documented in the specific sink document, in the table of parameters.